editor.planx.uk/src/@planx/components/Checklist/Editor/Options.tsx

```diff
@@ -28,6 +30,10 @@ export const Options: React.FC<{ formik: FormikHookReturn }> = ({ formik }) => {
 
   const exclusiveOrOptionManagerShouldRender =
     hasFeatureFlag("EXCLUSIVE_OR") && nonExclusiveOptions.length;
+  
+  const schema = useStore().getFlowSchema()?.options;
+  const initialOptions: Option[] | undefined = formik.initialValues.options || formik.initialValues.groupedOptions?.map((group: Group<Option>) => group.children)?.flat();
+  const initialOptionVals = initialOptions?.map((option) => option.data?.val);
```

Comment:
This is a bit of a lazy rebase right now while the exclusive-or work is still in-progress - I think `initialOptions` _should_ be able to be simplified/derived from `exclusiveOptions` or `nonExclusiveOptions` once those are handling "grouped" options